samples/pingpong/main.go

```diff
@@ -27,6 +28,12 @@ func main() {
 
 	// Get own ID from command line.
 	ownID := t.NodeID(os.Args[1])
+	// Get 0 for no debugger, 1 for debugger
```

Comment:
an optional `-d` as the first argument would be more intuitive to use. I.e.:
if the first argument is `-d`, use debug mode and look for the ID in the second argument. Otherwise, treat the first argument as the ID.
Alternatively, you could also use `cobra` (see other applications in the mir repo), but that still seems to be an overkill for this pingpong demo.